dist/rockcraft-pack-action/index.js

```diff
@@ -20117,13 +20119,14 @@ var RockcraftBuilder = class {
       rockcraftPackArgs = `${rockcraftPackArgs} --verbosity ${this.rockcraftPackVerbosity}`;
     }
     rockcraft = `${rockcraft} ${rockcraftPackArgs.trim()}`;
-    await exec3.exec(
-      "sudo",
-      ["--preserve-env", "--user", shellUser(), ...rockcraft.split(" ")],
-      {
-        cwd: this.projectRoot
-      }
-    );
+    let sourceCmd = "";
+    if (this.envFile) {
+      sourceCmd = `source ${this.envFile} && `;
+    }
+    const bashCommand = `${sourceCmd}sudo --preserve-env --user ${shellUser()} ${rockcraft}`;
```

Comment:
This is unrelated to the PR, but this is an interesting example of where switching to `sudo-rs` could cause issues.